d3a_api_client/rest_device.py

```diff
@@ -0,0 +1,163 @@
+from d3a_api_client import APIClientInterface
+import os
+import logging
+import requests
```

Comment:
Please add it in project requirements. Thanks!

Reply:
It's already there https://github.com/gridsingularity/d3a-api-client/blob/feature/D3ASIM-1943/requirements/base.in#L4

Reply:
strange, dont know why my PyCharm is still complaining